src/components/contribute/QuickstartResources.tsx

```diff
@@ -12,33 +13,25 @@ export default function QuickstartResources() {
       <View width={{ base: '90%', large: '60%' }}>
         <Heading level={2}>Quickstart videos</Heading>
         <Text fontSize="large">
-          Follow along with these videos to learn how to set up your local environment to get started making open source contributions to the Amplify project.
+          Follow along with these videos to learn how to set up your local
+          environment to get started making open source contributions to the
+          Amplify project.
         </Text>
       </View>
       <Flex wrap={'wrap'} justifyContent={'center'} width="100%" gap="2em">
         <View width={{ base: '90%', large: '40%' }}>
-          <iframe
-            sandbox
+          <YoutubeEmbed
+            src="https://www.youtube-nocookie.com/embed/8BUSqSkhqtw"
             width="600"
             height="350"
-            src="https://www.youtube-nocookie.com/embed/8BUSqSkhqtw"
-            title="YouTube video player"
```

Comment:
It looks like some of these properties are being lost in the transition over to the new component `frameborder` and `title`,  are these properties either no longer supported or needed?  Are there other properties that we want to be able to pass down to the player, how do we feel about just passing all props through?